Automattic/jetpack
# Admin: update upgrade links to use new instertials when possible

#### Changes proposed in this Pull Request:

﻿We've introduced new in-admin screens to promote upgrades; let's use them instead of directly linking to the checkout screen in Calypso, when possible. It's worth noting that for some of our upgrades (ads / Google Analytics), we do not have dedicated screens yet, so we'll continue to link to Calypso for those.

This was discussed in p8oabR-Nm-p2#comment-5914

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* You'll want to start with a free site, connected to your WordPress.com account.
* Go to Jetpack > Settings
    * Click the upgrade buttons available under Performance > Search, Performance > VideoPress, Security.
    * You should be redirected to an interstitial inside of the dashboard instea of being redirected to Calypso.
